q2cli/commands.py

```diff
@@ -225,6 +225,17 @@ def __init__(self, name, plugin, action):
                               'during execution of this action. Or silence '
                               'output if execution is successful (silence is '
                               'golden).'),
+            click.Option(['--parsl'], is_flag=True, required=False,
+                         help='Indicate that you want to execute your action '
+                              'with parsl. This flag will check the following '
```

Comment:
This comment isn't finished because we discussed the exact locations and the order in which it checks them potentially being in flux. We also discussed putting the default config in a vendored file, but I'm not sure exactly where we would want that to be.